app/controllers/restaurants_controller.rb

```diff
@@ -6,6 +6,7 @@ class RestaurantsController < ApplicationController
   def index
     @restaurants = Restaurant.all
     gon.restaurants = @restaurants
+    @select_options = Restaurant::PERMITTED_CATEGORIES.dup.insert(0, 'All Restaurants')
```

Comment:
Using `insert` changed the categories throughout the app. I have no idea why. But it broke tests. SO said to use `dup` and it works. (Creates a copy of the array)